rfcs/0109-nixpkgs-generated-code-policy.md

```diff
@@ -0,0 +1,210 @@
+---
+feature: nixpkgs-generated-code-policy
+start-date: 2021-10-12
+author: John Ericson (@Ericson2314)
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: @L-as @sternenseemann @tomberek @DavHau
+shepherd-leader: @sternenseemann
```

Comment:
```suggestion
author: @sternenseemann 
co-authors: @Ericson2314
shepherd-team: @L-as @kfearsoff @tomberek @DavHau
shepherd-leader: @tomberek
```
Apologies for the previous suggestion, where I missed out making sternenseemann the author.

Reply:
Wait, I thought @Ericson2314 wrote this?

Reply:
https://github.com/NixOS/rfcs/pull/109#issuecomment-1576424589